utils/tfhe-versionable/examples/transparent_then_not.rs

```diff
@@ -0,0 +1,162 @@
+//! This example is similar to the "transparent" one, except that the wrapper type is transparent at
+//! a point in time, then converted into its own type that is not transparent.
+//!
+//! To make this work, we consider that the inner and the wrapper type share the same history up to
+//! the version where the transparent attribute has been removed.
+
+use std::convert::Infallible;
+
+use tfhe_versionable::{Unversionize, Upgrade, Version, Versionize, VersionsDispatch};
+
+// This type was transparent before, but it has now been transformed to a full type.
+#[derive(Versionize)]
+#[versionize(MyStructWrapperVersions)]
+enum MyStructWrapper<T> {
+    OldWrapper(MyStruct<T>),
+    NewWrapper(u64),
+}
+
+// We need to create a dispatch enum that starts at the last version that was transparent
```

Comment:
That sentence confuses me, to me, the last version that was transparent is V1, not V0, so if I follow the `that starts at the last version that was transparent` the enum should start at V1

Reply:
Yeah I'm confused too, I will reword that